### DISCUSSION TITLE
1.21.2袭击机制再次更改

### DISCUSSION THREAD

**Discussion by vocaloidkiss** [Crystal0404/CrystalCarpetAddition#15] [Q&A] [answered]
cca恢复旧版袭击是否能生效？

**Comment by Crystal0404** [comment, marked as answer]
有效

---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年10月24日 22:15 |
| 收件人 | Crystal0404/CrystalCarpetAddition ***@***.***> |
| 抄送人 | Subscribed ***@***.***> |
| 主题 | [Crystal0404/CrystalCarpetAddition] 1.21.2袭击机制再次更改 (Issue #14) |

cca恢复旧版袭击是否能生效？

**Comment by Crystal0404** [comment]
另外请注意, 问题追踪器只用于bug报告与增强建议(虽然我没时间为cca增加新功能). 类似于这样的问题请使用讨论模块

**Reply by vocaloidkiss** [reply]
欧克欧克，没用过哈哈哈哈
